scripts/anvil_tools/README.md

```diff
@@ -253,3 +253,17 @@
     1. `--tsv`, `-t`: input .tsv file (required)
     2. `--array_columns`, `-a`: .txt file, new line delimited, to capture columns/attributes that are or array type (default = NO array type columns/attributes)
     3. `--json_output`: parameter to set if a local json file of the final json request is required (default = NO local json output file created)
+
+
+#### **examine_gcs_contents.py**
+##### Description
+    Outputs the contents of up to two GCS paths, comparing if multiple paths are specified.
```

Comment:
This is minuscule and obvious if you read the comments in the script - but it might be nice to specify here in the readme that you are comparing the contents, not the paths themselves.